Add fix to serialize inner model

jonatak opened the pull request:
Hello!

Nice project, I just start to experiment with it, might I suggest this fix?

This is to be able to serialize a model of this form:

```
class Other(Record):
    field: str

class Model(Record):
    other_record: Other
```

This code used for this fix is from here:

https://github.com/marcosschroh/python-schema-registry-client/blob/f18d0929f45bd5e096150db614f3bbec28346557/schema_registry/serializers/faust_serializer.py#L50

philipp94831 commented on faust_avro_serializer/avro_serializer.py:
```diff
@@ -25,7 +25,30 @@ def _loads(self, s: bytes) -> typing.Optional[typing.Dict]:
+        All values inside payload should be native types and not faust.Record
+        On Faust versions <=1.9.0 Record.to_representation always returns a dict with native types
+        as a values which are compatible with fastavro.
+        On Faust 1.10.0 <= versions Record.to_representation always returns a dic but values
```
Typo: dict

philipp94831 commented on faust_avro_serializer/avro_serializer.py:
```diff
@@ -25,7 +25,30 @@ def _loads(self, s: bytes) -> typing.Optional[typing.Dict]:
+    @staticmethod
+    def clean_payload(payload: typing.Dict[str, typing.Any]) -> typing.Dict[str, typing.Any]:
+        """
+        Try to clean payload retrieve by faust.Record.to_representation.
```
Typo: retrieved

philipp94831 commented on faust_avro_serializer/avro_serializer.py:
```diff
@@ -25,7 +25,30 @@ def _loads(self, s: bytes) -> typing.Optional[typing.Dict]:
+        Try to clean payload retrieve by faust.Record.to_representation.
+        All values inside payload should be native types and not faust.Record
+        On Faust versions <=1.9.0 Record.to_representation always returns a dict with native types
+        as a values which are compatible with fastavro.
```
Typo: value

philipp94831 commented on faust_avro_serializer/avro_serializer.py:
```diff
@@ -25,7 +25,30 @@ def _loads(self, s: bytes) -> typing.Optional[typing.Dict]:
+        On Faust versions <=1.9.0 Record.to_representation always returns a dict with native types
+        as a values which are compatible with fastavro.
+        On Faust 1.10.0 <= versions Record.to_representation always returns a dic but values
+        can also be faust.Record, so fastavro is incapable of serialize them
```
Typo: is incapable to serialize them

jonatak commented:
@philipp94831 Thanks for the review, I addressed your comments, and also expand the usecases:
ref: https://github.com/marcosschroh/python-schema-registry-client/pull/60
